recipes/fork-purger/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "fork-purger" %}
+{% set version = "21.1.3" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/fork_purger-{{ version }}.tar.gz
+  sha256: df4de6a12fe05f292b44bbd9cf27cecb070e78389327d2dd3d912106a70a074f
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - fork-purger = purger.main:cli
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+  run:
+    - anyio
+    - click >=7
+    - httpx >=0.16.0
```

Comment:
It looks like `httpx` is specified as `httpx[http2]>=0.16.0` in the upstream `setup.cfg`, which would require `h2`:
https://github.com/encode/httpx/blob/master/setup.py#L66-L67

I'm not sure if you need to add that as a dependency since conda doesn't really handle extras.

Reply:
If I get rid of it pip check will likely complain, but I can get rid of the pip check if that's more preferable? In other packages sometimes I just leave it in

Reply:
Just to be clear, I'm asking if you should add `h2` as an additional dependency, not remove `httpx` or the `pip check`.

Reply:
I'm unsure as well; what would you suggest?

Reply:
Given this code:

https://github.com/rednafi/fork-purger/blob/main/purger/main.py#L35

you are going to get an exception if `h2` is not available:

https://github.com/encode/httpx/blob/fa03b489f0fcd0df53afa8f9c471e24949f09e3e/httpx/_client.py#L637-L644

so I would add `h2` under `run` in the recipe.

Reply:
ok I'll do that